external/common/include/zconf.h

```diff
@@ -284,7 +284,7 @@ typedef uLong FAR uLongf;
    typedef Byte       *voidp;
 #endif
 
-#if 1           /* HAVE_UNISTD_H -- this line is updated by ./configure */
+#ifdef HAVE_UNISTD_H     /* HAVE_UNISTD_H -- this line is updated by ./configure */
```

Comment:
> /* HAVE_UNISTD_H -- this line is updated by ./configure */

It seems this line was updated by make/configure during zlib build. With CMake this line is not updated longer so this manual change was necessary.